Sources/NIOSSH/Connection State Machine/Operations/AcceptsUserAuthMessages.swift

```diff
@@ -16,6 +16,9 @@ protocol AcceptsUserAuthMessages {
     var userAuthStateMachine: UserAuthenticationStateMachine { get set }
 }
 
+/// This event indicates that server accepted our response to authentication challenge. SSH session can be considered active after that.
+public struct UserAuthSuccessEvent {}
```

Comment:
Can we add a `public init` to this? Otherwise users can't construct instances of it. We should also make it `Hashable`.

Reply:
done!